Makefile

```diff
@@ -3,6 +3,7 @@ SRCS_JS_FROM_TS =
 
 SRCS_TS += index.ts
 SRCS_TS += $(shell find lib -type f -name '*.ts')
+SRCS_TS += $(shell find test -type f -name '*.ts')
```

Comment:
Can we make this that we only compile tests when executing the test target?  I can assist with the Makefile structure if it helps.